app/components/shared/supplemental_files_step_body_table_component.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render Elements::Tables::TableComponent.new(id: 'supplemental-files-table',
+                                                label: 'Supplemental Files',
+                                                show_label: false) do |component| %>
+  <% component.with_headers([
+                              { label: 'Supplemental File (maximum size 1GB)' },
+                              { label: 'Type' },
+                              { label: 'Size' },
+                              { label: 'Upload Date' },
+                              with_remove ? { label: 'Remove', classes: 'sr-only' } : { label: '' }
+                            ]) %>
+  <% supplemental_files.each do |supplemental_file| %>
+    <% component.with_row do |row_component| %>
+      <%= row_component.with_cell do %><%= link_to(supplemental_file.filename, rails_blob_path(supplemental_file)) %><% end %>
+      <%= row_component.with_cell do %>PDF<% end %>
```

Comment:
Supplements aren't always PDFs.

Reply:
Good catch, I meant to change that. Fixed.